### ISSUE TITLE
Show The Title Of The Windows In Windows Manager

### ISSUE THREAD

**Issue by Archie-2021** [brookhong/Surfingkeys#1769]
I always name my windows, show the title of the windows in Windows Manager. Also it would be awesome if you add a shortcut for naming windows.

**Comment by daanzu** [comment]
Showing these window names in the window manager (like when we choose a window to move the current tab to) would be a great feature addition.

**Comment by Archie-2021** [comment]
> Showing these window names in the window manager (like when we choose a window to move the current tab to) would be a great feature addition.

Yeah , using move/w command is the only reason I bother to name Chrome/Brave Windows. Never seen another reason to do so.

**Comment by daanzu** [comment]
Well, I use https://github.com/daanzu/Switcheroo for switching between windows (of any windows application). But having it for moving tabs would be great help for organizing all of my tabs between all of my windows, both of which I have a pathological number.

**Comment by Archie-2021** [comment]
> Well, I use https://github.com/daanzu/Switcheroo for switching between windows (of any windows application). But having it for moving tabs would be great help for organizing all of my tabs between all of my windows, both of which I have a pathological number.

Cool , didn't know such a thing exist on Windows. There is a Rofi feature on Linux to do this and I even have set a hotkey for it , but I just switch between workspaces and just don't minimize windows. But I can see how this can be useful on Window.
